julearn/pipeline/merger.py

```diff
@@ -5,16 +5,20 @@
 
 from typing import Dict
 
-from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
+from sklearn.model_selection._search import BaseSearchCV
```

Comment:
Is there a way to get around not using the class from a private file?

Reply:
No. They don't expose the BaseSearchCV